azclishell/configuration.py

```diff
@@ -111,4 +115,16 @@ def get_config_dir():
         return os.path.expanduser(os.path.join('~', '.azure-shell'))
 
 
+def ask_user_for_telemetry():
```

Comment:
The shell itself doesn't need to ask for telemetry. See message above and try out the CLI experience also.